### PR TITLE
Add description, url and repositories to project

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 (defproject witan.models.demography "0.1.0-SNAPSHOT"
-  :description "FIXME: write description"
-  :url "http://example.com/FIXME"
+  :description "witan.models.demography is a Clojure library to run demographic models."
```
Comment (sunnytownsend) [inline project.clj:2]: common mistake to say demography models rather than demographic models, so nice one @acron0

**Comment by sunnytownsend** [comment]
:shipit:

**Comment by sunnytownsend** [comment]
once tests pass of course ...

File: project.clj (hunk 1 above)
Comment (acron0) [inline project.clj:2]: I copied this from the README :sunglasses:

**Pull request merged**